docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,32 @@
+### UPDATE
+
+If a ``WHERE`` clause is specified, the ``UPDATE`` operation only works if the
+predicate in the clause can be fully pushed down to the data source.
+
+Only ``UPDATE`` statements with constant assignments and predicates are
+supported. For example, the following statement is supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1
+```
+
+Arithmetic expressions can not be used with the ``SET`` command. The following
+statement is not supported:
+
+```sql
+UPDATE table SET col1 = col2 + 2 WHERE col3 = 1
+```
+
+``AND`` and ``OR`` operators can not be used in predicates. The following
+statement is not supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1 AND col2 = 3
+```
+
+All column values of a table row can not be updated simultaneously. The
```

Comment:
All? 

You mean not more than one?

Reply:
it could be more then one but not all columns : 
like if have 6 columns in table we can update 5, 4, ... simultaneously, but not 6

Reply:
I meant all. You can update multiple columns simultaneously. The distinction is that you can't update every column in the table simultaneously.